### ISSUE TITLE
Re-add slide up effect on the modal for mobile

### ISSUE THREAD

**Issue by rolznz** [getAlby/bitcoin-connect#132]
It was accidentally removed when the modal animation was removed for both desktop and mobile. The modal design is different (where the modal is attached to the bottom of the screen, and a slide up is a nice effect.

**Comment by Nomishka** [comment]
@rolznz Is this issue still available?

**Comment by rolznz** [comment]
@Nomishka that's correct. There was some code here but I think it was also applied to desktop at that time. We should use the proper tailwind breakpoint (max:sm? to apply the animation) https://github.com/getAlby/bitcoin-connect/blob/fb2670d66b3b7debfbd40c930971393e1c624586/src/components/lwc-modal.ts#L13

**Comment by Nomishka** [comment]
@rolznz I solved this issue with the pull request https://github.com/getAlby/bitcoin-connect/pull/265. Can you please check it? Please let me know if you need further adjustments. 😊

**Comment by Nomishka** [comment]
Hello @rolznz, I would like to remind you about my pull request #265 related this issue. Can you please review it when possible? Please let me know if you need further adjustments. :-)

**Comment by rolznz** [comment]
@Nomishka thanks for the reminder. I will try to review it this week

**Comment by Nomishka** [comment]
Thank you @rolznz! Appreciate it :-)
